consensus/istanbul/validator/default.go

```diff
@@ -222,7 +222,7 @@ func (valSet *defaultSet) SubListWithProposer(prevHash common.Hash, proposer com
 	subset := make([]istanbul.Validator, valSet.subSize)
 	subset[0] = New(proposer)
 	// next proposer
-	subset[1] = valSet.selector(valSet, subset[0].Address(), view.Round.Uint64()+1)
+	subset[1] = valSet.selector(valSet, subset[0].Address(), view.Round.Uint64())
```

Comment:
How about updating parameters of the selector method?
`func roundRobinProposer(valSet istanbul.ValidatorSet, curProposer common.Address, curRound uint64)`